src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.ComposeStore.targets

```diff
@@ -211,6 +220,10 @@ Copyright (c) .NET Foundation. All rights reserved.
       <SkipUnchangedFiles Condition="'$(SkipUnchangedFiles)' == ''">true</SkipUnchangedFiles>
     </PropertyGroup>
 
+   <NETSdkError Condition="'2.0' > '$(_TargetFrameworkVersionWithoutV)'"
+                 ResourceName="NU1008"
+                 FormatArguments=".NET Core $(_TargetFrameworkVersionWithoutV)"/>
```

Comment:
Maybe this should pass the whole `$(TargetFrameworkIdentifier)` in instead of hard-coding `.NET Core`.